pre-docs/writing_plans.md

```diff
@@ -132,59 +132,13 @@ Your plan can execute multiple functions on remote systems.
 
 Your plan can include functions to run commands, scripts, tasks, and other
 plans on remote nodes. These execution functions correspond to task runner
-commands.
-
-- `run_command`: Runs a command on one or more nodes.
-- `run_script`: Runs a script (a non-task executable) on one or more nodes.
-- `run_task`: Runs a task on one or more nodes.
-- `run_plan`: Runs a plan on one or more nodes.
-- `file_upload`: Uploads a file to one or more nodes.
-
-
-### Calling basic plan functions
-
-Basic functions in plans share a similar structure. Call these functions with their parameters.
-
-`run_script`
-Runs a script on one or more nodes.
-`$fileref`, `$nodes`, `$description`, `$options`
-
-`file_upload`
-Uploads a file to a specified location on one or more nodes. Note that most transports are not
-optimized for file copying, so this is best limited to small files.
-`$source`, `$destination`, `$nodes`, `$description`, `$options`
-
-`run_command`
-Runs a command on one or more nodes.
-`$cmd`, `$nodes`, `$description`, `$options`
-
-`get_targets`
-Parses common ways of referring to targets and returns an Array of Target objects.
-`TargetSpec $targetspec`
-
-For the functions `run_script` and `file_upload`, the `$fileref` and `$source`
-parameters accept either an absolute path or a module relative path, `<MODULE
-NAME>/<FILE>` reference, which will search for `<FILE>` relative to a module's
-files directory. For example, the reference `mysql/mysqltuner.pl` searches for
-the file `<MODULES DIRECTORY>/mysql/files/mysqltuner.pl`.
-
-The `$options` parameter is used for options that modify how Bolt executes the function. For `run_script`, arguments to the script can be passed as an array of strings under an `arguments` key in `$options`.
+commands. For a complete list of functions that can be run from plans see the
+[plan function reference
+documentation](https://github.com/puppetlabs/bolt/blob/master/doc/index.html).
```

Comment:
I did the full url here since I'm assuming that's what'll get copied to the docs. But if we copy the rendered markdown, I think this can be relative.

Reply:
I think we're going to be creating a new docs page in puppet.com/docs.

Reply:
These all seem to depend on checking in code generated by https://github.com/puppetlabs/bolt/pull/473.

Reply:
Yep -- I was planning to wait until your PR was merged before generating the docs, though I can do it now (shrug)

Reply:
Yeah, I think it makes sense for each set of docs to have internal links.